br/pkg/lightning/backend/local/iterator_test.go

```diff
@@ -237,6 +245,6 @@ func BenchmarkDupDetectIter(b *testing.B) {
 		for iter.First(); iter.Valid(); iter.Next() {
 			keyCnt++
 		}
-		iter.Close()
+		require.NoError(b, iter.Close())
```

Comment:
I remember when testing external/bench_test.go, require.NoError is slow and occupy a bit in the CPU flame graph if called too frequently. Because this is inside the benchmark part, I prefer we use intest.AssertNoError